controllers/case-controller.js

```diff
@@ -75,7 +81,25 @@ const CaseController = Case => {
     await caseFromDB.destroy();
   };
 
-  return { createCase, getCase, listCases, updateCase, deleteCase };
+  const searchCases = async (body, query) => {
+    if (isEmptySearchParams(body)) {
+      return { count: 0, contacts: [] };
+    }
+    console.log({ query });
+    const [cases, metadata] = await sequelize.query(searchCasesQuery, {
+      replacements: {
+        helpline: body.helpline || null,
+        firstName: body.firstName ? `%${body.firstName}%` : null,
+        lastName: body.lastName ? `%${body.lastName}%` : null,
+        dateFrom: body.dateFrom || null,
+        dateTo: body.dateTo || null,
+        phoneNumber: body.phoneNumber ? `%${body.phoneNumber}%` : null,
```

Comment:
Do we need to strip non-digits out of the phone number here as in https://github.com/tech-matters/hrm/blob/master/controllers/contact-controller.js#L99 ?

Reply:
Also while I'm here, I'll mention to @GPaoloni that the hardcoded fields in https://github.com/tech-matters/hrm/blob/master/controllers/contact-controller.js#L111 may be a problem for customization.  Nothing to do on that for right now, just a thing to remember.